wordpress-mobile/gutenberg-mobile
# Prevent caching of `video` endpoint on Android

* `gutenberg`: https://github.com/WordPress/gutenberg/pull/50216

## Description

As outlined in https://github.com/WordPress/gutenberg/pull/50216, a new filter, `native.disabled_caching_endpoints`, has been introduced that can dynamically extend the list of endpoints that shouldn't be cached on Android devices. 

This PR utilises that changes in order to add the `/rest\/v1.1\/videos.*/i` endpoint to the `DISABLED_CACHING_ENDPOINTS`. This means any requests made to this endpoint will not be cached, which fixes the issue described [here](https://github.com/wordpress-mobile/WordPress-Android/pull/18327#issuecomment-1526228675) with VideoPress settings being reset on Android.

Note, support for the `/rest\/v1.1\/videos.*/i` endpoint was added in https://github.com/wordpress-mobile/gutenberg-mobile/pull/5652 and it's currently only used for VideoPress requests, as such no regressions are expected to emerge from this change. 

## Testing

* In the Android app, navigate through the steps to add the VideoPress block to the editor.
* Choose the option to upload a video from your device.
* Open the settings panel to edit the video's title and description.
* Exit the editor.
* Re-open the editor and open the settings panel.
* Verify that the updated title and description remain and _don't_ reset.
* Go through some other general flows related to the VideoPress block's settings (e.g. updating then saving) to verify there are no other regressions. 

<hr />

PR submission checklist:
